ui/revenuecatui/src/main/res/values/paywall_view_attrs.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <declare-styleable name="PaywallFooterView">
+        <attr name="android:fontFamily" />
+        <attr name="offeringIdentifier" format="string" />
```

Comment:
These are the parameters that can be passed from the xml. This is limited, some of them can only be passed through the view.